schemas/view-schema.json

```diff
@@ -219,6 +219,64 @@
                 "readOnly": true
               },
               "readOnly": true
+            },
+            "metric": {
+              "type": "object",
+              "description": "Metrics of the view",
+              "properties": {
+                "memory": {
+                  "type": "object",
+                  "description": "Metrics related to the virtual memory used by the view",
```

Comment:
"Used" is misleading (we can't guess which of pre-allocated memory was used). I think "allocated" is what we really mean, no?